<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>innertab</class>
 <widget class="QWidget" name="innertab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1011</width>
    <height>603</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>981</width>
     <height>591</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QPushButton" name="addpushButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deletepushButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="texteditcheckBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>普通文本</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="markdowncheckBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>130</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>markdown</string>
   </property>
   <property name="tristate">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
